src/plot_styles.py

```diff
@@ -411,35 +391,20 @@ def save_style(self):
         file = directory.get_child_for_display_name(f"{style.name}.mplstyle")
         file_io.write_style(file, style)
 
-    def delete_color(self, _, color_box):
+    def delete_color(self, _button, color_box):
```

Comment:
Indeed these `_` arguments were the one I was talking about leading to issues in the current build. In general we should probably get rid of all of these _ arguments to avoid future problem.